### PR TITLE
Supporting the topological logging manager via launch files

### PR DESCRIPTION
**Pull request by cdondrup**
This adds the appropriate parameter to the launch files to use the topological logging manager or not.

Also now comparing for the empty string for the topic instead of None because there is no None.

### CODE REVIEW

**Comment by cdondrup** [comment]
Since we are now following the 4 eyes are better than two approach, @lucasb-eyer can you please have a look at this PR and #159 since it also touches on your packages.

File: vision_people_logging/launch/logging_ubd.launch [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <launch>
+    <arg name="manager" default=""/>
```
Comment (lucasb-eyer) [inline vision_people_logging/launch/logging_ubd.launch]: Could you rename to `manager_topic` like in the other cases, please?

**Comment by lucasb-eyer** [comment]
There is no `None` [for me either](https://github.com/strands-project/strands_perception_people/blob/indigo-devel/vision_people_logging/scripts/save_ubd.py#L36) then, could you please change that in the same PR too?

**Comment by lucasb-eyer** [comment]
Also, sorry, the second pair of eyes will be slow in the next three weeks because they're getting hit by the other project.

**Comment by cdondrup** [comment]
@lucasb-eyer thanks for checking. Will make the changes you asked for and commit before merging.

**Pull request merged**